service/Core/DataFormats/Office/MsPowerPointDecoder.cs

```diff
@@ -1,80 +1,55 @@
-// Copyright (c) Microsoft. All rights reserved.
+﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using DocumentFormat.OpenXml.Packaging;
 using DocumentFormat.OpenXml.Presentation;
+using Microsoft.Extensions.Logging;
+using Microsoft.KernelMemory.Diagnostics;
+using Microsoft.KernelMemory.Pipeline;
 
 namespace Microsoft.KernelMemory.DataFormats.Office;
 
-public class MsPowerPointDecoder
+public class MsPowerPointDecoder : IContentDecoder
 {
-    private readonly string _slideNumberTemplate;
-    private readonly string _endOfSlideMarkerTemplate;
-
-    /// <param name="slideNumberTemplate">Template used for the optional slide number added at the start of each slide</param>
-    /// <param name="endOfSlideMarkerTemplate">Template used for the optional text added at the end of each slide</param>
-    public MsPowerPointDecoder(
-        string slideNumberTemplate = "# Slide {number}",
-        string endOfSlideMarkerTemplate = "# End of slide {number}")
+    private readonly MsPowerPointConfig _config;
+    private readonly ILogger<MsPowerPointDecoder> _log;
+
+    public IEnumerable<string> SupportedMimeTypes { get; } = new[] { MimeTypes.MsPowerPointX, MimeTypes.MsPowerPoint };
```

Comment:
why add MimeTypes.MsPowerPoint here if the format is not supported and the result is an exception?

Reply:
Because I have kept the original behavior that uses a specific log message for old Office 97-2003 file formats: https://github.com/microsoft/kernel-memory/blob/489e4c66f96b0f3f4320d140a46d69fa65a1a26a/service/Core/Handlers/TextExtractionHandler.cs#L173. If it is OK to use the standard message, then we can safely remove this.